hr_employee_legacy_id/__manifest__.py

```diff
@@ -21,7 +21,7 @@
 
```

Comment:
@GillesTephaneMeyomesse Thanks for the update of the module, can you please revise the files with latest module template, meaning short headers in special...